README.md

```diff
@@ -22,9 +22,12 @@ sudo apt-get install texlive-xetex
 # initaite a new contract
 claw new john examples/templates/consultant.md
 
+# change into the newly generated directory
+cd john
+
 # edit the params
-vim john/params.toml
+vim params.toml
 
 # compile the markdown and output a final pdf using pandoc
-claw compile --output pdf john
+claw compile --output pdf
```

Comment:
the command `claw compile` probably expects two commands so the code should be amended to reflect that.